modes/config-mode/public/theme/theme.json5

```diff
@@ -4,14 +4,25 @@
 
   ConfigurableModes: {
     modes: {
-      "@ohif/longitudinal": "@ohif/mode-longitudinal",
+      "hpMode": "hpMode",
       "ecgMode": { configOperation: "reference", source: "ecgMode"},
       "mprMode": { configOperation: "reference", source: "mprMode"},
       "microscopyMode": { configOperation: "reference", source: "microscopyMode"},
+      "@ohif/mode-tmtv": "@ohif/mode-tmtv",
     },
     clearDefaultModes: true,
   },
 
+  hpMode: {
```

Comment:
The hpMode defined here replaces the basic/original viewer mode, and can be further extended with extra default settings.